### ISSUE TITLE
css src write after html src filename do not be replaced

### ISSUE THREAD

**Issue by dukai** [hollandben/grunt-cache-bust#156]
```
cacheBust: {
  options: {
  },
  assets: {
    files: [{   
      expand: true,
      cwd: '<%= config.dist %>',
      src: ['page/**/*.html' ,'css/**/*.css']
    }]
  }
}
```
When I write config like this, the image file names in style sheets files do not be replaced, but when I write like below it works:
```
cacheBust: {
  options: {
  },
  assets: {
    files: [{   
      expand: true,
      cwd: '<%= config.dist %>',
      src: ['css/**/*.css', 'page/**/*.html']
    }]
  }
}
```

**Comment by hollandben** [comment]
That's a strange bug. Could you try using the `v1.0.0-beta` branch and see if that fixes the problem. After a few people have tested the branch I'll get it merged in.

**Comment by dukai** [comment]
I downloaded the v1.0.0-beta,but it did not work.
Cannot read property 'indexOf' of undefined

**Comment by hollandben** [comment]
Thanks for trying. Was that the complete error given back? Did it give you back a stack trace?

**Comment by hollandben** [comment]
Possibly fixed when fixing #173 - needs re-testing

**Comment by hollandben** [comment]
I've tested this myself and can no longer see the issue. Let me know if it comes back
